cacti/tests/README.md

```diff
@@ -0,0 +1,21 @@
+To run integration tests you need to install python-rrdtool
+
+Install python-rrdtool on MAC
+=============================
+```
+brew install rrdtool
+pip install rrdtool
+```
+
+To run integration tests
+========================
+* `ddev env start cacti py27-integration`
+* `docker ps`
+*  Install required dependencies in agent image:
+    ```
+    $ docker exec -it <agent_image> /bin/bash
```

Comment:
Any way we could automate this?

Reply:
Since the name of the agent container should be fixed, maybe we could do some `run_command(['docker', 'exec', ...])` in the `check` fixture